README.md

```diff
@@ -25,11 +25,12 @@ npm run build
 
 ## Deploying
 
-To deploy this app to Github pages:
+The app is hosted through AWS. New commits on the main branch will trigger a rebuild of the container images in our ECR repository. (See the [workflow](https://github.com/nahmii-community/bridge/blob/bf4c0b5736be77e34e7d7ee0c41b76fa3b4c7d25/.github/workflows/aws-ecr.yaml) file for more details)
```

Comment:
Almost but not quite. It doesn't trigger a rebuild of any extisting images. How about:

New commits to the main brance are automatically built as a docker image. Docker tag corresponds to git hash.
The latest version is automatically deployed to staging. In order to update the deployed version in production: <the steps below>

Reply:
Updated the text according to your feedback @arvearve. Thank you. :)